__tests__/app.test.js

```diff
@@ -148,16 +148,37 @@ describe("GET", () => {
         });
     });
 
+
+    //task-11
+    it("200: responds with an array of articles that can be sorted by any of its properties passed as a query and in descending order as default ", () => {
+      return request(app).get('/api/articles?sort_by=created_at&order=desc').expect(200).then((result) => {
+        expect(result.body.articles).toBeSortedBy('created_at' , {descending : true})
+      })
+    })
```

Comment:
Please check another combination of sort and order as I think that this is the default 🙂